draft-nuttycom-funding-allocation.rst

```diff
@@ -0,0 +1,609 @@
+===========================================================================
+
+The following requirements apply to Alternative 2 and Alternative 4:
+
+The stream allocated to Zcash Community Grants (ZCG) is intended to fund
+independent teams entering the Zcash ecosystem, to perform major ongoing
+development (or other work) for the public good of the Zcash ecosystem, to the
+extent that such teams are available and effective. The ZCG Committee is given
+the discretion to allocate funds not only to major grants, but also to a
+diverse range of projects that advance the usability, security, privacy, and
+adoption of Zcash, including community programs, dedicated resources, and other
+projects of varying sizes.
+
+The funds SHALL be received and administered by the
+Financial Privacy Foundation (FPF). FPF MUST disburse them for grants and
+expenses reasonably related to the administration of the ZCG program, but
+subject to the following additional constraints:
+
+1. These funds MUST only be used to issue grants to external parties that are
+   independent of FPF, and to pay for expenses reasonably related to the
+   administration of the ZCG program. They MUST NOT be used by FPF for
+   its internal operations and direct expenses not related to the
+   administration of grants or the grants program.
+
+2. ZCG SHOULD support well-specified work proposed by the grantee, at
+   reasonable market-rate costs. They can be of any duration or ongoing without
+   a duration limit. Grants of indefinite duration SHOULD have semiannual
+   review points for continuation of funding.
+
+3. Priority SHOULD be given to major grants that bolster teams with substantial
+   (current or prospective) continual existence, and set them up for long-term
+   success, subject to the usual grant award considerations (impact, ability,
+   risks, team, cost-effectiveness, etc.). Priority SHOULD be given to major
+   grants that support ecosystem growth, for example through mentorship,
+   coaching, technical resources, creating entrepreneurial opportunities, etc.
+   If one proposal substantially duplicates another’s plans, priority SHOULD be
+   given to the originator of the plans.
+
+4. The ZCG committee SHOULD be restricted to funding projects that further the
+   Zcash cryptocurrency and its ecosystem (which is more specific than
+   furthering financial privacy in general) as permitted by FPF
+   and any relevant jurisdictional requirements.
+
+5. ZCG awards are subject to approval by a five-seat ZCG Committee. The ZCG
+   Committee SHALL be selected by the ZF’s Community Advisory Panel or a
+   successor process (e.g. as established by FPF). Elections SHALL be staggered
+   to ensure continuity within the Committee.
+
+6. The ZCG Committee’s funding decisions will be final, requiring no approval
+   from the FPF Board, but are subject to veto if the FPF judges them to
+   violate any relevant laws or other (current or future) obligations.
```

Comment:
This section is AFAICT largely unaltered from ZIP 1014, other than replacing ZF with FPF. There are parts of this section about FPF that are [noticeably different and/or more specific in the ZF ZIP draft](https://zips.z.cash/draft-zf-community-dev-fund-2-proposal#zcg-slice-zcash-community-grants). I'm not going to make any changes in this editorial pass; instead I will open a subsequent PR that aligns this ZIP draft with the ZF one, in order to make the differences between them clear (and so the owners of this ZIP draft can consider those differences individually; there is no requirement for the two ZIP drafts to actually align here, other than as needed by FPF itself).

Reply:
I spoke to Alex from ZF/FPF today. I am going to review the precise language in the ZF draft over the next couple days, discuss it with Alex and then we'll come back to you. Thanks.